### PR TITLE
0.8.0 release

### PR DESCRIPTION
**Pull request by rmlarose**
Description
-----------

See the changelog for a summary of this release.

Checklist
-----------

Check off the following once complete (or if not applicable) after opening the PR. The PR will be reviewed once this checklist is complete and all tests are passing.

- [ ] I added unit tests for new code.
- [ ] I used [type hints](https://www.python.org/dev/peps/pep-0484/) in function signatures.
- [ ] I used [Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) docstrings for functions.
- [ ] I [updated the documentation](../blob/master/docs/CONTRIBUTING_DOCS.md) where relevant.

If some items remain, you can mark this a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).

Tips
----

- If the validation check fails:

    1. Run `make check-style` (from the root directory
  of the repository) and fix any [flake8](http://flake8.pycqa.org) errors.

    2. Run `make format` to format your code with the [black](https://black.readthedocs.io/en/stable/index.html)
  autoformatter.

  For more information, check the [style guidelines](https://mitiq.readthedocs.io/en/stable/contributing.html#style-guidelines) for Mitiq.
  
- Write "Fixes #XYZ" in the description if this PR fixes Issue #XYZ.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,46 @@
 % # " - [Bug Fix]"
 % # " - Fix the bug."
 
+## Version 0.8.0 (May 5th, 2021)
```
Comment (nathanshammah) [inline CHANGELOG.md:11]:
```suggestion
## Version 0.8.0 (May 6th, 2021)
```

**Review by nathanshammah — APPROVED** [review]
Hi Ryan, I just made a single comment to update the date. However I approved already. Remember not to "squash and merge" but just merge with all commits.

File: VERSION.txt [hunk 2]
```diff
@@ -1 +1 @@
-0.7.0
+0.8.0dev
```
Comment (nathanshammah) [inline VERSION.txt:1]:
```suggestion
0.8.0
```

**Comment by nathanshammah** [comment]
The version number needs to be updated. Also the pdf is not updated but I guess we should just drop it in lieu of the RTD one.

**Comment by rmlarose** [comment]
Dropping the PDF, will close this and re-open a new release PR.

**Pull request closed without merging**